pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -1052,25 +1055,37 @@ func getTaskrunWorkspaces(ctx context.Context, pr *v1beta1.PipelineRun, rpt *res
 	return workspaces, pipelinePVCWorkspaceName, nil
 }
 
-// taskWorkspaceByWorkspaceVolumeSource is returning the WorkspaceBinding with the TaskRun specified name.
-// If the volume source is a volumeClaimTemplate, the template is applied and passed to TaskRun as a persistentVolumeClaim
-func taskWorkspaceByWorkspaceVolumeSource(wb v1beta1.WorkspaceBinding, taskWorkspaceName string, pipelineTaskSubPath string, owner metav1.OwnerReference) v1beta1.WorkspaceBinding {
+// taskWorkspaceByWorkspaceVolumeSource is returning the WorkspaceBinding for TaskRuns.
+// If the WorkspaceBinding a volumeClaimTemplate, an associated existing persistentVolumeClaim is binded to the TaskRun,
+// otherwise, the PipelineRun WorkspaceBinding is applied and binded to the TaskRun.
```

Comment:
```suggestion
// taskWorkspaceByWorkspaceVolumeSource returns the WorkspaceBinding to be bound to each TaskRun in the Pipeline Task.
// If the PipelineRun WorkspaceBinding is a volumeClaimTemplate, the returned WorkspaceBinding references a PersistentVolumeClaim created for the PipelineRun WorkspaceBinding.
// Otherwise, the returned WorkspaceBinding references the same volume as the PipelineRun WorkspaceBinding.
```

Reply:
It would also be helpful to document the "owner" and "pipelineTaskSubPath" arguments.

Reply:
String doc updated and included documentation for "owner" and "pipelineTaskSubPath" arguments.